### PR TITLE
Make multiple checked checkboxes only return one answer as a list; test ...

### PR DESCRIPTION
**Pull request by moduloprime**
...that post-processing script handles these OK

Addresses #206 by making multiple checked checkboxes only return one response, with a comma-separated list of all checked answers, instead of separate response per checked answer.

@adrifelt for Javascript part
@meacer for Python

### CODE REVIEW

File: extension/surveys/survey-generator.js [hunk 1]
```diff
@@ -542,11 +542,12 @@ function getFormValues(questionArr, form) {
...
-      for (var j = 0; j < answerList.length; j++) {
-        var response = new SurveySubmission.Response(
-            questionStr, answerList[j].value || 'NOANSWER');
-        responses.push(response);
+      var answer = answerList[0].value || 'NOANSWER';
+      for (var j = 1; j < answerList.length; j++) {
+        answer = answer.concat(",", answerList[j].value)
       }
+      var response = new SurveySubmission.Response(questionStr, answer);
+      responses.push(response);
```
Comment (adrifelt) [inline extension/surveys/survey-generator.js]: this change LGTM, but it makes me realize: what's happening to "Other" answers here? are we completely losing them in the responses (both before and after this change)?

**Comment by moduloprime** [comment]
In my testing (and looking at other responses), I got "2-Other-Whatisyourgender", so we get the Other, but not the actual text associated with it.

**Comment by adrifelt** [comment]
Filed a bug for that: https://github.com/GoogleChrome/experience-sampling/issues/222

But how would you like it to be reported to the backend? There may be things like commas in it, so we can't concatenate.

File: tools/processresults_test.py [hunk 2]
```diff
@@ -12,29 +12,35 @@
...
                      'and Internet-related items? I have...(DendoPort)'),
                     csv_headers)
       self.assertIn(('How familiar are you with each of the following computer '
                      'and Internet-related items? I have...(TCP/IP)'),
                     csv_headers)
-      
+
       test_line = csv_file.readline()
-      self.assertIn('P2DA1,P2DendoPortAnswer,P2TCPIPAnswer', test_line)
+      print test_line
```
Comment (meacer) [inline tools/processresults_test.py]: Did you mean to keep the print?

File: tools/processresults.py [hunk 3]
```diff
@@ -23,7 +24,9 @@
 ATTRIBUTE_QUESTION_PREFIX = 'To what degree do each of the following'
 
 
-def ProcessResults(json_in_file, csv_prefix):
+def ProcessResults(json_in_file, csv_prefix,
+                   start_date = FRIENDS_AND_FAMILY_BETA_DATE,
```
Comment (meacer) [inline tools/processresults.py]: nit: Could |start_date| be renamed to |filter_items_before_date| or something like that? Same for and |demographic_start_date|.

**Comment by meacer** [comment]
Python lgtm.

File: tools/processresults_test.py (hunk 2 above)
Comment (moduloprime) [inline tools/processresults_test.py]: Good catch; removed it.

File: tools/processresults.py (hunk 3 above)
Comment (moduloprime) [inline tools/processresults.py]: Done.

**Pull request merged**